### ISSUE TITLE
Pylance should be able to recognize docstrings for type aliases

### ISSUE THREAD

**Issue by macintacos** [microsoft/pylance-release#1815]
I can't seem to get VSCode to recognize docstrings for type aliases that I define. Say I have the following type alias and docstring:

```python
SomeType = List[Union[dict, str]]
"""Here's some documentation about SomeType"""
```

For an example of how I'd like it to work, Pycharm shows the following when I hover over SomeType:

<img width="417" alt="CleanShot PyCharm 2021-09-13 at 17 44 33" src="https://user-images.githubusercontent.com/8128941/133160744-1c9981e4-cd55-40c5-8d62-7b7784c83192.png">

In VSCode I only seem to see documentation related to the underlying types instead of the documentation I provided for the alias:

<img width="967" alt="CleanShot Code 2021-09-13 at 17 46 57" src="https://user-images.githubusercontent.com/8128941/133160846-d98e499c-6c8f-44a6-aca8-abc1c6a4f4da.png">

In an ideal world, I'd be able to see both the underlying type documentation _and_ the documentation that I added for the alias myself, but honestly I'd settle for at least always showing the documentation that I specified in the project itself.

Can this be added? This felt like something that should be implemented in Pylance, but I wasn't 100% sure - please feel free to move it to whatever project makes the most sense!

**Comment by jakebailey** [comment]
Interesting; this should have worked as we explicitly intend to support these types of docstrings (see #1576).

![image](https://user-images.githubusercontent.com/5341706/133164575-a92e3632-e4f8-4c15-b29b-ebf5972a3e29.png)

This may be something specific to aliases.

**Comment by erictraut** [comment]
We do support variable doc strings, but type aliases are distinct from other types of variables. It should be feasible to extend the support for type aliases.

**Comment by jakebailey** [comment]
Yep, it's a small change (I'm making it now).

**Comment by jakebailey** [comment]
In this case, it's hitting the fact that it's an instantiable type before it hits the code that checks if it's a variable.

**Comment by jakebailey** [comment]
This will be fixed in the next release.

**Comment by macintacos** [comment]
That's great, thanks!

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.9.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202192-16-september-2021
